filecheck/checker.ts

```diff
@@ -269,29 +269,60 @@ function canCheckFile(filePath: string) {
 async function resolveDirectory(file: string): Promise<string[]> {
   const stats = await fs.lstat(file);
   if (stats.isDirectory()) {
-    const api = new fdir()
-      .withErrors()
-      .withFullPaths()
-      .filter((filePath) => canCheckFile(filePath))
-      .crawl(file);
+    const api = new fdir().withErrors().withFullPaths().crawl(file);
```

Comment:
We still need to filter irrelevant files here:

1. Files outside of `files`.
2. Files inside `node_modules`.
3. Files named `.DS_Store`.

So maybe those conditions could be extracted from `canCheckFile()` into a separate `isRelevantFile()` function.

Reply:
Agree. Instead of checking for each unwanted folder we could check for only the two required paths. And for files we need to ignore `.DS_Store` (mac) and `.ini`(Windows).
```js
function isRelevantFile(filePath: string) {
  return (filePath.includes(`files${path.sep}en-us`) || filePath.includes(`files${path.sep}jsondata`)) &&
    !/\.(DS_Store|ini)$/i.test(filePath);
}
```
This way we won't have to worry about every new directory added by IDEs/plugins/operating systems. For example, `.code`, `.idea`, and `.husky`.